clients/admin-ui/src/features/privacy-requests/privacy-requests.slice.ts

```diff
@@ -514,8 +514,8 @@ export type CORSOrigins = Pick<SecurityApplicationConfig, "cors_origins">;
  * be consistent!
  */
 export type CORSOriginsSettings = {
-  configSet: Pick<SecurityApplicationConfig, "cors_origins">;
-  apiSet: Pick<SecurityApplicationConfig, "cors_origins">;
+  configSet: SecurityApplicationConfig & { cors_origin_regex?: string };
+  apiSet: SecurityApplicationConfig;
```

Comment:
SecurityApplicationConfig now only has the `cors_origins` field, so these picks were redundant.